plugins/sdk-python/example/Dockerfile.test

```diff
@@ -40,4 +40,4 @@ COPY ./plugins/sdk-python/example ./src
 USER 65534
 
 # Run example scanner
-CMD ["python3", "/src/scanner.py"]
+CMD ["/usr/local/bin/python3", "/src/scanner.py"]
```

Comment:
To use the binary plugin support it's a requirement to provide the full path of the binary in the image of the plugin container image. (this is also advantageous from a security point of view)

This is a limitation that comes from the Go's exec package: https://cs.opensource.google/go/go/+/refs/tags/go1.22.4:src/os/exec/exec.go;drc=f7f266c88598398dcf32b448bcea2100e1702630;l=418

Whether you run a command in a chroot or not, Go will attempt to resolve the path of the binary based on the PATH set before the chroot.